### PR TITLE
Merge with latest version of spec.

### PR DESCRIPTION
**Pull request by g-ortuno**
There were no merge conflicts. The only change that affects our implementation is the removal of BluetoothDiscovery, BluetoothInteraction, BluetoothUUIDs, etc.

Patch to remove those from implementation:

http://crrev.com/1145323002

### CODE REVIEW

File: charter.md [hunk 1]
```diff
@@ -0,0 +1,184 @@
+# Web Bluetooth Community Group Charter
```
Comment (scheib) [inline charter.md]: The merge should not include this file.

File: implementation-status.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
+# Implementation Status
```
Comment (scheib) [inline implementation-status.md]: The merge should not include this file.

**Comment by scheib** [comment]
LGTM after the few files that shouldn't merge are excluded.

File: charter.md (hunk 1 above)
Comment (g-ortuno) [inline charter.md]: Done.

File: implementation-status.md (hunk 2 above)
Comment (g-ortuno) [inline implementation-status.md]: Done.

**Pull request merged**